### ISSUE TITLE
[Bug]: Address accessibility concern with sp-tabs-overflow component

### ISSUE THREAD

**Issue by monkeyjabs** [adobe/spectrum-web-components#2955]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

sp-tabs-overflow

### Expected behavior

**sp-tabs-overflow** component should pass accessibility unit test

### Actual behavior

**sp-tabs-overflow** is failing on accessibility unit test

```
Error: Accessibility Violations
      ---
      Rule: aria-command-name
      Impact: serious
      ARIA commands must have an accessible name (https://dequeuniversity.com/rules/axe/4.4/aria-command-name?application=axeAPI)
      
      Issue target: x-home-quick-actions-row,sp-tabs-overflow,.right-scroll
      Context: <sp-action-button quiet="" class="right-scroll show" dir="ltr" size="m" role="button" focusable="" tabindex="0">
                          <sp-icon-chevron100 slot="icon" class="spectrum-UIIcon-ChevronRight300" dir="ltr"></sp-icon-chevron100>
                      </sp-action-button>
      Fix any of the following:
        Element does not have text that is visible to screen readers
        aria-label attribute does not exist or is empty
        aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
        Element has no title attribute
      ---
      Rule: scrollable-region-focusable
      Impact: moderate
      Scrollable region must have keyboard access (https://dequeuniversity.com/rules/axe/4.4/scrollable-region-focusable?application=axeAPI)
      
      Issue target: x-home-quick-actions-row,sp-tabs,#list
      Context: <div id="list" role="tablist" part="tablist" class=" scroll ">
                      <slot></slot>
                      <div id="selection-indicator" role="presentation" class="first-position" style="transform: translateX(0px) scaleX(0) scaleY(0)"></div>
                  </div>
      Fix any of the following:
        Element should have focusable content
        Element should be focusable
      ---

```

### How can we reproduce this issue?

3. Scroll to '....'
4. Check console
5. See error

**Comment by Westbrook** [comment]
fixed by #4165
